website_sale_vat_required/static/src/js/website_sale_vat_required.tour.js

```diff
@@ -1,40 +1,40 @@
 /* License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
  */
-
 odoo.define('website_sale_vat_required.tour', function(require) {
-
     "use strict";
 
-    require("website_sale.tour");
-    var base = require("web_editor.base");
     var tour = require("web_tour.tour");
     var steps = tour.tours.shop_buy_product.steps;
 
     steps.splice(
-        _.findIndex(steps, { 'content': "Confirm checkout"}),
+        _.findIndex(steps, { 'content': "go to checkout"}) + 1,
         0,
         {
             content: 'Next',
-            trigger: ".btn-primary:contains('Next')",
+            trigger: ".btn-primary :contains('Next')",
         },
         {
             content: 'Set VAT',
-            trigger: "div.has-error input[name='vat']",
-            extra_trigger: "div.has-error input[name='vat']",
+            trigger: "div.o_has_error input[name='vat']",
+            extra_trigger: "div.o_has_error input[name='vat']",
             run: function (actions) {
-                $('div.has-error input[name="vat"]').val("BE0477472701");
-                if ($('#div_phone').hasClass('has-error')){
+                $('div.o_has_error input[name="vat"]').val("BE0477472701");
+                if ($('#div_phone').hasClass('o_has_error')){
                     $('#div_phone input').val('11111111');
                 }
-                var input_accept_legal_terms = $('div.has-error input[name="accepted_legal_terms"]');
+                var input_accept_legal_terms = $('div input[name="accepted_legal_terms"]');
```

Comment:
I'm not sure about this change. How are you checking that it has an error then?

Reply:
I already changed it and left it as it was before